### ISSUE TITLE
[Feature] High MAR from Transaction Accounting Line

### ISSUE THREAD

**Issue by fivetran-jamie** [fivetran/dbt_netsuite#49]
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Describe the Feature

A few customers have noted the `transaction_accounting_line` source table is a high MAR table and would prefer to disable this model if possible. This feature is intended to allow a variable that disables the `transaction_accounting_line` model without having an impact on the downstream Netsuite models.

> **Note**: The `transaction_accounting_line` source table is a crucial component of our transaction line calculation. This is then used in all downstream financial report models. We will not be able to do a simple disable model variable as that would result in unusable downstream models. We will need to find another approach. See the "Anything else?" section of this FR for more details.

### Describe alternatives you've considered

Not sync the source table in question and result in failures in the dbt run as the package requires this source.

### Are you interested in contributing this feature?

- [X] Yes.
- [ ] Yes, but I will need assistance and will schedule time during your [office hours](https://calendly.com/fivetran-solutions-team/fivetran-solutions-team-office-hours) for guidance.
- [ ] No.

### Anything else?

Based on the [dbt_netsuite DAG](https://fivetran.github.io/dbt_netsuite/#!/model/model.netsuite.netsuite2__transaction_details?g_v=1&g_i=%2Bstg_netsuite2__transaction_accounting_lines%2B) the `stg_netsuite2__transaction_accounting_lines` model is used in our downstream models. In fact, it is a crucial piece to capturing the total amounts for each transaction line. See our [int_netsuite2__transaction_lines](https://github.com/fivetran/dbt_netsuite/blob/f3a559948182615e4259c0b106813faca8880457/models/netsuite2/intermediate/base/int_netsuite2__transaction_lines.sql) model where this model is used to generate the totals which are used downstream in the greater financial reports.
![image](https://user-images.githubusercontent.com/74217849/190662440-e893ee40-f274-450e-9156-42e4669c4929.png)

Due to the importance of this source, I am curious how the package would be effective with its removal. A better alternative from disabling the source would be to find a way to allow for the totals of the transaction lines to be generated from somewhere else. However, that does not seem to be the case at the moment.

Also, this issue has a mirror issue in the source package to increase visibility 
https://github.com/fivetran/dbt_netsuite_source/issues/22

**Comment by fivetran-jamie** [comment]
Ok, so a huge driver of the high MAR was this table's inability to sync incrementally. There was no `last_modified_timestamp` column, so for each sync, every single row was resynced and recounted, even if nothing changed. This was due to a limitation on Netsuite's side that they have since resolved!! 🥳 🎉 🎈 

The `transaction_accounting_line` table will officially have a `last_modified_timestamp` to it in the upcoming Netsuite [2023.1 release](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/chapter_3715068484.html)! This release has already begun for some customers, however each customer has their own rollout from the Oracle side. Our connector will immediately begin syncing incrementally for this table once a customers table appears with this field.

tldr; This table is significantly less problematic if you upgrade to Netsuite [2023.1](https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/chapter_3715068484.html)
